src/main/scala/gpp/ui/forms/EnumSelect.scala

```diff
@@ -25,21 +25,21 @@ final case class EnumSelect[A](
   onChange:    A => Callback = (_: A) => Callback.empty
 )(implicit
   val enum:    Enumerated[A],
-  val show:    Show[A]
+  val display: Display[A]
 ) extends ReactProps[EnumSelect[Any]](EnumSelect.component)
 
 object EnumSelect {
   type Props[A] = EnumSelect[A]
 
   implicit protected def propsReuse[A]: Reusability[Props[A]] =
-    Reusability.by(p => (p.label, p.value.map(p.show.show), p.placeholder, p.disabled))
+    Reusability.by(p => (p.label, p.value.map(p.display.shortName), p.placeholder, p.disabled))
```

Comment:
While we are on this, can we try switching to the `enumReuse` from the `reusability` package?

Reply:
Sure. I also noticed that the other 2 selects don't have `Reusability`s. I assume they should?

Reply:
Yes, that would be great. Thank you.

Reply:
Should I remove this one, then? Or let it be?

Reply:
Never mind. Leaving be - I see there are fewer properties here.